cmake/modules/hunter_windows_utils.cmake

```diff
@@ -0,0 +1,43 @@
+# Copyright (c) 2013, 2015 Ruslan Baratov
+# All rights reserved.
+
+# CMAKE function to convert windows log filename to 8.3 filename
+function(hunter_windows_to_snf_filename path new_path_variable)
+  set(conversion_cmd
+    "cmd" 
+    "/c" 
+    "for" 
+    "%A" 
+    "in" 
+    ("${path}") 
+    "do" 
+    "@echo"
+    "%~sA"
+	)
+  
+  execute_process(
+    COMMAND ${conversion_cmd}
+    RESULT_VARIABLE conversion_result
+    OUTPUT_VARIABLE converted_path
+  )
+  
+  if(NOT conversion_result EQUAL 0)
+    hunter_fatal_error("Command to convert path " ${path} " to 8.3 Filename failed. Command is " ${conversion_cmd})
```

Comment:
We got 4 arguments to the `hunter_fatal_error` call here. It mean output will be:
```
[hunter ** FATAL ERROR **] Command to convert path
[hunter ** FATAL ERROR **] ${path}
[hunter ** FATAL ERROR **]  to 8.3 Filename failed. Command is
[hunter ** FATAL ERROR **] ${conversion_cmd}
```
if you want one line you can write:
```cmake
"Command to convert path ${path} to 8.3 Filename failed. Command is ${conversion_cmd}"
```